pkg/pillar/types/certinfotypes.go

```diff
@@ -6,22 +6,43 @@ package types
 import (
 	"encoding/hex"
 
-	zcert "github.com/lf-edge/eve/api/go/certs"
-	zcommon "github.com/lf-edge/eve/api/go/evecommon"
+	"github.com/google/go-tpm/tpmutil"
+	"github.com/lf-edge/eve/api/go/evecommon"
 )
 
-// ControllerCert : controller certicate
-// config received from controller
-type ControllerCert struct {
-	HashAlgo zcommon.HashAlgorithm
-	Type     zcert.ZCertType
-	Cert     []byte
-	CertHash []byte
-	// ErrorAndTime provides SetErrorNow() and ClearError()
+type ZCertOrigin uint8
+
+const (
+	CERT_ORIGIN_NONE ZCertOrigin = iota + 0
```

Comment:
We never use a Z prefix for types and values in EVE. The API uses it for some things, but if you define such in EVE it results in confusion.